pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,141 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file, gds_number, elevation=None, thickness=None, import_method=1):  # pragma: no cover
+        """Import a GDSII file.
+
+        Parameters
+        ----------
+        gds_file : str
+            Path to the GDS file.
+        gds_number : list
+            List of layer numbers to import.
+        elevation : list, optional.
+            List of GDS layer elevation.
+            Number of element must be the same as gds_number list, if not elevation and thickness will not be applied.
+            If elevation is "None", elevation and thickness will not be applied.
+            The default is ``None``
+        thickness : list, optional
+            List of GDS layer thickness.
+            Number of element must be the same as gds_number list, if not elevation and thickness will not be applied.
+            If thickness is "None", elevation and thickness will not be applied.
+            The default is ``None``
+        import_method : integer, optional
+            GDSII import method.
+            Options are:
+            - 0 = script
```

Comment:
put 0 within `` and 0 for script without the equal and same for next line

Reply:
- ``0`` for script

Reply:
done